libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.scss

```diff
@@ -271,7 +271,7 @@
       .sky-toggle-switch-switch {
         background-color: var(
           --sky-override-toggle-switch-background-color-disabled,
-          var(--sky-color-background-input-disabled)
+          var(--sky-color-background-selected-switch-disabled)
```

Comment:
Not sure if this is a here problem - or a token problem - or actually what they want - but it appears that the disabled toggle when it is "on" doesn't look much (if any) different than a normal enabled toggle.